lib/pysquared/battery_helper.py

```diff
@@ -52,7 +55,7 @@ def _wait_for_ack(self):
             if self.uart.in_waiting:
                 byte = self.uart.read(1)
                 if self.debug_mode:
-                    print(f"ACK byte received: {byte}")
+                    self.logger.info(message=f"ACK byte received: {byte}")
```

Comment:
There is nothing wrong with this but... now that `message` is the first argument, we could leave off the key here. That would make the call to the logger feel similar to `loguru` [link](https://loguru.readthedocs.io/en/stable/overview.html#ready-to-use-out-of-the-box-without-boilerplate). It's less work for consumers of the logger and they can choose in interact with kwargs if they need to.

What about something like this?
```suggestion
                    self.logger.info("ACK received", "ack"=byte)
```